styles/styles.css

```diff
@@ -267,23 +267,54 @@ blockquote {
   margin: var(--spacer-element-05) 0 var(--spacer-element-05);
 }
 
-main .column-content-top-border div>div {
+main .section > div {
+  width: 80vw;
+  margin: 0 auto;
+  }
+
+main .columns-gap-50 > div {
+  gap: 50px
```

Comment:
`;`

Reply:
Nice catch. I am also of the school to always use a semicolon, even on the last line. There are probably more of these from the existing code.